website/docs/components/form/masked-input/partials/guidelines/guidelines.md

```diff
@@ -68,6 +68,19 @@ For shorter, simpler forms, indicate optional fields instead.
   <F.Label>Label</F.Label>
 </Hds::Form::MaskedInput::Field>
 
+## Copyable masked input 
```

Comment:
Not overly opinionated here, but "Copyable" isn't really sitting right with me. Maybe consider changing this headline to something like "Copying the Masked Input" (also note the capitalization when the component name is being referenced). Some other options while I'm at it:

- Copying the Masked Input value
- Masked Input copying

Reply:
+1, I like "Copying the Masked Input value" personally. It's the most clear.